crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/execution/cairo1_execution.rs

```diff
@@ -91,7 +95,7 @@ pub fn execute_entry_point_call_cairo1(
         });
     }
 
-    Ok(call_info)
+    Ok((call_info, syscall_counter))
```

Comment:
The reason it's returned here is that blockifier no longer collects this along with the resources, so we have to do that manually.

Reply:
Isn't it better to drag it out in the `execute_call_entry_point`? I forgot about this part and was wondering if the returned counter was global or not

Reply:
Not possible since the syscall counter/handler is not created yet in this function